Code/Managers/DSPDistrictStylePackageManager.cs

```diff
@@ -116,14 +124,14 @@ internal static void RefreshDistrictStyleAssetsMetaData(DistrictStyle districtSt
             }
 
             dsMeta.assets = newAssetList;
-            StylesHelper.SaveStyle(dsMeta, dsMeta.name, false);
-            PackageManager.EnabledEvents();
```

Comment:
why is DisabledEvents removed?

Reply:
StylesHelper.SaveStyle already calls disabledEvents and enabledEvents. I assume that's enough, but of course it wouldn't hurt to call it too... I kept it as comment in line 205 and 218.

Not calling PackageManager.ForcePackagesChanged() might be more problematic. I honestly don't know whether it is needed or not. If this is too much work to test, no worries. I can play with this for a while longer and if there are still no problems send another PR...